addons/binding/org.openhab.binding.airquality/README.md

```diff
@@ -92,15 +87,15 @@ HAZARDOUS=Hazardous
 
 airquality.things:
 
-```
+```text
```

Comment:
Why are all of those "text"? I'd suggest anything other than text to get a colored highlighting. As with most openHAB DSL text, they look good with a "java" setting:

```javascript
 airquality:aqi:home "AirQuality" @ "Krakow" [ apikey="XXXXXXXXXXXX", location="50.06465,19.94498", refresh=60 ]
 airquality:aqi:warsaw "AirQuality in Warsaw" [ apikey="XXXXXXXXXXXX", location="52.22,21.01", refresh=60 ]
 airquality:aqi:brisbane "AirQuality in Brisbane" [ apikey="XXXXXXXXXXXX", stationId=5115 ]
```